Realm/RLMRealm.h

```diff
@@ -91,7 +93,7 @@
 
  @return An `RLMRealm` instance.
  */
-+ (instancetype)realmWithPath:(NSString *)path readOnly:(BOOL)readonly error:(NSError **)error;
++ (nullable instancetype)realmWithPath:(NSString *)path readOnly:(BOOL)readonly error:(NSError **)error;
```

Comment:
Per [WWDC 2015 session 401](https://developer.apple.com/videos/wwdc/2015/?id=401) (at 18m35s), `NSError**` parameters are assumed to be nullable for both levels within audited regions.